docs/revocation.md

```diff
@@ -1,9 +1,12 @@
-If the offline CA has been compromised, excluding the offline root CA, the following script should be run in a ceremony to revoke the intermediate offline CA's certificate:
-`templates/authenticated/revoke-certificate.sh`
+If a CA signed by an offline CA has been compromised (excluding the offline root CA), the script `revoke-certificate.sh` should be run in a ceremony to revoke the CA's certificate.
 
-You should then regenerate all children CAs (but not end device certificates)
+You should then regenerate all children of the compromised CA (but not end device certificates)
 
-If the online CA has been compromised, the following script should be run in a ceremony to revoke the offline CA's certificate:
-`templates/authenticated/revoke-certificate.sh`
+If the root CA has been compromised, then you need to contact anyone relying on it and warn them to stop trusting it.
```

Comment:
```suggestion
If the root CA has been compromised, then you need to contact all services that rely on it as a trusted CA so that they stop trusting it.
```